docs/upgrading.md

```diff
@@ -0,0 +1,66 @@
+# Upgrading Contour to 0.15
+
+## Changes in 0.15
+
+### Breaking change
+The major breaking change in Contour 0.15 is that `contour serve` (the mode that actually serves the control plane for Envoy) now requires that either TLS certificates be available, or you supply the `--insecure` parameter.
```

Comment:
s/The major breaking change//

Contour 0.15's `contour sever` command requires ...